services/src/test/java/org/keycloak/test/broker/oidc/AbstractOAuth2IdentityProviderTest.java

```diff
@@ -121,10 +132,54 @@ public void getFederatedIdentity_responseUrlLine() {
 		Assert.assertEquals("458rtf", fi.getId());
 	}
 
-	private TestProvider getTested() {
+	@Test
```

Comment:
Hi, in this test I was just validating that "performLogin" service, which create the authorization url is working properly after the change I have done.
That's why, here I have testes it with an idp configured with activated/deactivated toggle on loginHint.

Can you explain, or show an exemple of the kind of test you are waiting for ? 
Thx in advance